package.json

```diff
@@ -57,7 +57,7 @@
+    "redoc": "redoc-cli bundle $npm_package_config_specfile -t actions/redoc/template.hbs --options.hideDownloadButton=true --options.expandResponses='200,201' --templateOptions.segmentKey=0ysbW8vXHeRAnY2saZPMlRSZFfPwd1RO --templateOptions.SUPPORT_URL=https://support.lob.com/hc/en-us --templateOptions.LOB_ASSETS_URL=https://s3-us-west-2.amazonaws.com/public.lob.com --templateOptions.canonicalUrl=https://docs.lob.com/ --templateOptions.keywords=\"Direct Mail, Address Verification\" --templateOptions.description=\"Detailed developer documentation for Lob's address verification API and print & mail API. Learn how to easily integrate Lob's APIs within your product or technology stack.\" --templateOptions.LOB_URL=https://lob.com --templateOptions.image=https://s3-us-west-2.amazonaws.com/public.lob.com/marketing/opengraph2.png --templateOptions.DASHBOARD_URL=https://dashboard.lob.com/# --title=\"DRAFT NEW Lob API documentation\" --options.noAutoAuth=false --options.maxDisplayedEnumValues=15 --options.theme.colors.primary.main=#27333c --options.theme.sidebar.backgroundColor=#f7f9fa --options.jsonSampleExpandLevel=1 --options.requiredPropsFirst=true --options.theme.typography.code.backgroundColor=#ebf0f6 --options.theme.typography.code.fontFamily=JetBrainsMono --options.theme.typography.code.color=#27333c --options.theme.typography.fontFamily=\"Larsseit,sans-serif;\" --options.theme.typography.links.color=#0099d7 --options.theme.typography.links.textDecoration=none --options.theme.codeBlock.backgroundColor=#ecf0f5 --options.theme.rightPanel.backgroundColor=#f7f9fa --options.theme.rightPanel.textColor=#416581 --options.theme.typography.fontSize=16 -o docs/index.html",
```

Comment:
We have 2 authentication sections now. Is this what you intended?

Reply:
Sadly - there is a link with each operation that points to the default "authentication" section and I can't find a way to remove the link or change it - so I added the default authentication section back in and renamed ours "basic authentication".  If you find a way around this please let me know.

Reply:
I think I could include that in the remapping-hashes PR so u can probably get rid of the default "authentication" section and turn "basic authentication" back into "authentication". And I can map `#section/Authentication/basicAuth` to `#tag/Authentication`

Reply:
I've also hidden the download button as I have no idea how it works, but it someone generates a file called swagger.json for download.  It's cool an all, but we are using OpenAPI not swagger so it's confusing.

We could create our own download button and link to our deployment branch of the lob-openapi repo in the future.